### PR TITLE
[FEATURE] implemented shipping order dependency for shippings

### PR DESCRIPTION
**Pull request by turbo-ele**
**Tasks:**
- [x] implemented order dependency class 
- [x] function for count shippings by order id
- [x] gather feedback

**Informations:**

| Q | A |
| --- | --- |
| Tests pass? | - |
| Fixed tickets | - |
| BC Breaks | - |
| Doc | - |

### CODE REVIEW

File: Dependencies/ShippingOrderDependencyClass.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    }
+
+    /**
+     *
+     * @param $orderId
+     * @return array
+     */
+    public function getDocuments($orderId)
+    {
+        $documents = array();
```
Comment (michaelzangerle) [inline Dependencies/ShippingOrderDependencyClass.php:89]: @turbo-ele return always an empty array?

File: Entity/ShippingRepository.php [hunk 2]
```diff
@@ -148,6 +148,34 @@ public function getSumOfShippedItemsByOrderId($orderId)
     }
 
     /**
+     * returns number of shippings by order id
+     * @param $orderId
+     * @param array $statusIds
+     * @param string $comparator
+     * @return int|mixed
+     */
+    public function countByOrderId($orderId, $statusIds = array(), $comparator = "=")
```
Comment (michaelzangerle) [inline Entity/ShippingRepository.php:157]: @turbo-ele would it make sense to have for each status a comparator?

File: Dependencies/ShippingOrderDependencyClass.php (hunk 1 above)
Comment (turbo-ele) [inline Dependencies/ShippingOrderDependencyClass.php:89]: currently, yes. its prepared for you to implement it ;)

Comment (michaelzangerle) [inline Dependencies/ShippingOrderDependencyClass.php:89]: then add a comment please :)

**Comment by michaelzangerle** [comment]
ack

File: Entity/ShippingRepository.php (hunk 2 above)
Comment (turbo-ele) [inline Entity/ShippingRepository.php:157]: actually, yes. but currently it's not needed;

**Pull request merged**